packages/config/config/devices/0x010f/fgt001.json

```diff
@@ -28,22 +37,121 @@
 			"maxValue": 10000,
 			"defaultValue": 240
 		},
-		"2": {
-			"label": "Additional functions",
-			"description": "This parameter allows to enable different additional functions of the device.",
+		"2[0x01]": {
+			"label": "Enable open window detector (normal)",
+			"$import": "../templates/master_template.json#base_enable_disable",
```

Comment:
`$import` should come before labels and such, otherwise the label might get overwritten.